### PR TITLE
feat: adds reusable workflows for cra

### PR DESCRIPTION
**Pull request by fitimbyttyqi**
[issues-28/cra-reusable-actions](https://github.com/ornio-no/issues/issues/28)

### CODE REVIEW

**Comment by fitimbyttyqi** [comment]
@oxodesign I need to make further tests with cra reusable workflows, tomorrow we could make a meeting or maybe give me access to merge instead of waiting for you because you are busy to always make approves.
If we keep a meeting I would like to know some things with im unclear with regarding the setup with cra applications

File: .github/workflows/cra-build.branches.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: CRA Build Branches
```
Comment (oxodesign) [inline .github/workflows/cra-build.branches.yml:1]: @fitimbyttyqi ❌ file name should be cra-build-branches.yml (you have a . instead of -)

**Review by oxodesign — CHANGES_REQUESTED** [review]
@fitimbyttyqi ❌ minor file name issue, see the comment otherwise great job :)

**Comment by fitimbyttyqi** [comment]
@oxodesign Done :)

**Review by oxodesign — APPROVED** [review]

**Pull request merged**